Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && \
+    apt-get install -y wget \
+                       curl \
+                       software-properties-common \
+                       git \
+                       unzip \
+                       postgresql \
+                       default-jdk
```

Comment:
If you need `jdk` you can use one of the dsapp `docker` images located in the `ECR`